sky/tools/create_macos_framework.py

```diff
@@ -34,6 +34,7 @@ def main():
   parser.add_argument('--x64-out-dir', type=str, required=True)
   parser.add_argument('--strip', action="store_true", default=False)
   parser.add_argument('--dsym', action="store_true", default=False)
+  parser.add_argument('--zip', action="store_true", default=False)
```

Comment:
https://github.com/flutter/flutter/issues/110374#issuecomment-1228989160
> I'll send a PR protecting the creation of the zip file with a flag that will be used only on the recipes v2 builds.

Is that what this is? Maybe a comment and todo to remove once it's adoped?

Reply:
TODO added.